docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -145,115 +166,116 @@ can set the following in your ``galaxy.yml`` file:
      - '*.tar.gz'
 
 .. note::
-     This feature is only supported when running ``ansible-galaxy collection build`` with Ansible 2.10 or newer.
+     The ``build_ignore`` feature is only supported with ``ansible-galaxy collection build`` in Ansible 2.10 or newer.
 
-.. _collection_versions:
+.. _trying_collection_locally:
 
-Collection versions
-===================
+Preparing to publish your collection
+====================================
 
-Each time you publish your collection, you create a new version. Once you publish a version of a collection, you cannot delete or modify that version. Ensure that everything looks okay before publishing. The only way to change a collection is to release a new version. The latest version of a collection (by highest version number) will be the version displayed everywhere in Galaxy or Automation Hub; however, users will still be able to download older versions.
+Each time you publish your collection, you must create a :ref:`new version <collection_versions>` on the distribution server. After you publish a version of a collection, you cannot delete or modify that version. To avoid unnecessary extra versions, check your collection for bugs, typos, and other issues locally before publishing:
```

Comment:
```suggestion
Each time you publish your collection, you must create a :ref:`new version <collection_versions>` on the distribution server. After you publish a version of a collection, you cannot delete or modify that version. To avoid unnecessary extra versions, check your collection for bugs, typos, and other issues locally before publishing.
```

Reply:
Thanks for the detailed comments and suggestions. In my mind, the `:` connects the paragraph to the two list items below. With a `.` there instead, the connection feels less clear. I prefer the flow with the `:`.